frontend/src/features/contacts/contact/detail/utils.ts

```diff
@@ -0,0 +1,89 @@
+import { IContactAddress } from 'interfaces/IContact';
+
+export const sortAddresses = (a1: IContactAddress, a2: IContactAddress) => {
+  if (a2.addressType.id === a1.addressType.id) {
+    return 0;
+  }
+
+  if (a2.addressType.id === 'MAILING') {
+    return 1;
+  }
+
+  if (a2.addressType.id === 'RESIDNT' && a1.addressType.id !== 'MAILING') {
+    return 1;
+  }
+  return -1;
+};
+
+export const fakeAddresses = [
```

Comment:
If this is for testing only, lets move it to a testing file instead of just util